templates/homepage-catalyst-band.tmpl

```diff
@@ -0,0 +1,26 @@
+<div class="grid-wrapper homepage-catalyst-band full-width-band"
+     id="catalyst">
+  <div class="section-title width-12-12 width-12-12-m">
+    <div class="section-top">
+      <img class="section-icon"
+           src="{{ homepage.catalyst.section_icon }}"
+           alt="{{ homepage.catalyst.icon_alt }}"
+           width="40"
+           height="40" />
+      <h2>{{ homepage.catalyst.title }}</h2>
+    </div>
+    <p>{{ homepage.catalyst.intro }}</p>
+  </div>
+  {% for key, item in homepage.catalyst.articles.items() %}
+    <div class="catalyst-card width-6-12 width-12-12-m">
+      <div class="catalyst-content">
+        <h3 style="color: red;">{{ item.heading }}</h3>
+        <p>{{ item.description }}</p>
+        <h5>{{ item.data_title }}</h5>
```

Comment:
From the accessibility report this skips a heading level. It goes from h3 to h5.

Reply:
The styling in place for heading was not consistent, so instead of changing them I just used the one that worked for the purpose needed.